media/Config/pathAutoComplete.js

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const getInputPath = function(tool) {
```

Comment:
Just a comment) if all the JS files run in flat space (that is all variables are global) so that `getInputPath` can be visible in other files JS files, then as `getInputPath` is generic name that may be used in other files.
If this happens, we will not know when somthing strange happens (when there are several developers working on different files and don't have time to look on each others codes...)

I think there are some solutions to solve this as JS has long history...
Could you consider this and revise this codes later?